pkgs/development/tools/build-managers/goredo/fix-tests.diff

```diff
@@ -0,0 +1,36 @@
+diff -ur goredo-1.4.1/t/apenwarr/105-sympath/all.do goredo-1.4.1.new/t/apenwarr/105-sympath/all.do
+--- goredo-1.4.1/t/apenwarr/105-sympath/all.do	2021-04-29 14:03:11.000000000 +0200
++++ goredo-1.4.1.new/t/apenwarr/105-sympath/all.do	2021-05-10 15:04:46.912799930 +0200
```

Comment:
```suggestion
+++ goredo-1.4.1/t/apenwarr/105-sympath/all.do	2021-05-10 15:04:46.912799930 +0200
```
Shouldn't this be like the above to apply correct?

Reply:
The first path element is irrelevant, because patches are usually applied with `-p1` (for that very reason).